error_checks.c

```diff
@@ -5,35 +5,38 @@
 #include "mcc_generated_files/system/system.h"
 
 #include "error_checks.h"
-#include "board.h"
+// #include "board.h"
 #include "actuator.h"
 
+const float CONVERT_FACTOR = 1e+05 / 4096.0f * 3.3; // 100 V/V multiplier * mV conversion * (raw + continuity correction) / 12bit adc * vref
```

Comment:
This isn't correct. An adc raw output of 1 = 1/4096 * 3.3V into adc = 1/4096 * 3.3V / 100 across resistor, this is the inverse of that (and idk where the 1e5 came from). Also remember when you divide by a resistance in milliohms, you end up with mV / mohm = A, not mA! So you want to multiply the conversion factor by 1000 so you have microvolts.